db/migrations/20190329104428_create_frob_event_computed_columns.sql

```diff
@@ -3,10 +3,14 @@
 
 -- Extend frob_event with ilk_state
 CREATE FUNCTION api.frob_event_ilk(event api.frob_event)
-    RETURNS api.ilk_state AS
+    RETURNS api.historical_ilk_state AS
```

Comment:
Probably missing something but I can't see where in the migrations the `api.historical_ilk_state` is defined 🤔 

I see it in the schema but wanting to double check that the relevant migration (with the name change) was checked in

Reply:
Yeah, `historical_ilk_state` is the name of the trigger table. 

https://github.com/makerdao/vdb-mcd-transformers/blob/vdb-1064-replace-ilk-query/db/migrations/20190301141224_create_ilk_state_history_table_and_triggers.sql#L2